common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3488,4 +3499,78 @@ object CelebornConf extends Logging {
       .doc("Whether to use codegen for columnar-based shuffle.")
       .booleanConf
       .createWithDefault(false)
+
+  // Flink specific client configurations.
+  val CLIENT_MEMORY_PER_PARTITION_MIN: ConfigEntry[Long] =
+    buildConf("celeborn.client.flink.partition.minMemory")
+      .withAlternative("remote-shuffle.job.min.memory-per-partition")
+      .categories("client")
+      .version("0.3.0")
+      .doc("Min memory reserved for a partition. For Flink plugin only.")
```

Comment:
"For Flink plugin only." is not needed then

Reply:
> the configuration "doc" is too rough
> 
> ```
> buildConf("celeborn.client.flink.inputGate.memory")
>       .doc("Memory reserved for a input gate. For Flink plugin only.")
>       ...
> ```
> 
> For example, the above "doc" does not help, users can not gain more information from "doc" statement than the configuration key itself. "input gate" is a new concept, it should be explained briefly at least in one of the keys' doc.

The concepts "input gate" and "result partition" are defined in Flink if a user uses Celeborn for flink. There is no reason to assume that the user is not familiar with Flink. So the explanation is unnecessary.